pkg/node/requester.go

```diff
@@ -84,6 +90,42 @@ func NewRequesterNode(
 		}),
 	)
 
+	// evaluation broker
+	evalBroker, err := evaluation.NewInMemoryBroker(evaluation.InMemoryBrokerParams{
+		VisibilityTimeout:    config.EvalBrokerVisibilityTimeout,
+		InitialRetryDelay:    config.EvalBrokerInitialRetryDelay,
+		SubsequentRetryDelay: config.EvalBrokerSubsequentRetryDelay,
+		MaxReceiveCount:      config.EvalBrokerMaxRetryCount,
+	})
+	if err != nil {
+		return nil, err
+	}
+	evalBroker.SetEnabled(true)
```

Comment:
yes, but it can be stopped and started multiple times in its lifecycle. We don't have that today, but will come handy when we introduce leader election where only a single eval broker can be enabled